process.go

```diff
@@ -95,9 +95,13 @@ type ProcessConfigurer interface {
 	// have a unique name.
 	Name(n string)
 
-	// RouteEventType configures the engine to route events of the same type as m
-	// to the handler.
-	RouteEventType(m Message)
+	// AcceptsEventType configures the engine to route events of the same type
```

Comment:
This one differs slightly in wording to the others by not mentioning "messages". Not that this is a problem, but just thought I would mention it.